src/bandersnatch/main.py

```diff
@@ -152,6 +158,15 @@ def main():
         finally:
             loop.close()
     else:
+        if args.force_check:
+            status_file = os.path.join(config.get("mirror", "directory"), "status")
+            try:
+                shutil.move(status_file, "/tmp/status")
+                logger.debug(
+                    "Force bandersnatch to check everything against the master PyPI."
+                )
+            except Exception as e:
+                logger.debug("Could not move status file: {}".format(str(e)))
```

Comment:
I feel if someone wants this to happen and it does not, this is an error ...
```suggestion
                logger.error("Could not move status file: {}".format(str(e)))
```

Reply:
You're right. The reason I chose debug is, I though these message makes the program ugly, since they appears before anything. I though maybe it's better to show them only when a user wants to debug the program.